### PR TITLE
comment out debug logs

### PR DESCRIPTION
**Pull request by Stefie**
The debug logs that spit out the complete hex-encoded metadata multiple times make it quite hard to follow the tests and bump up the time the tests are running significantly without adding any value, unless someone is actually updating the metadata.

If needed, the logs can be easily reactivated.

### CODE REVIEW

**Comment by Stefie** [comment]
I changed it only comment out those logs that produce a lot of noise

**Comment by Stefie** [comment]
> right now, NODE_ENV=abc yarn test ... runs tests without any noise. We could always change this logic to only show noise on let's say DEBUG=debug yarn test, but I don't think adding/removing debug lines manually on each test is the way to go.

I didn't know that! I'm really for setting the default to "no noise'" since you really only need 'some'(!) of the metadata log when you're working with the metadata directly and never all of them.

As opposed to that, running tests is the default behaviour when developing and testing all branches and PR's and the logs really slow that down.

**Comment by amaurymartiny** [comment]
So we can close this? It's a polkadot/common issue

**Comment by jacogr** [comment]
Yes, doesn’t make sense at all as is. 

Not sure about the “log slowdown” either - if you want to inspect specific, run those tests only or switch of logging completely with the NODEENV option - “debug” always logs and that will not change on common. 

(Yes, they can be very noisy and provide too much info - but as somebody who has probably run yarn test as much as anybody, not seeing what this means to address)

**Comment by jacogr** [comment]
Closing.
